src/ansys/additive/core/parametric_study/parametric_study.py

```diff
@@ -1670,38 +1662,108 @@ def _test_single_bead_input(self, row: pd.Series) -> tuple[bool, str]:
         Returns
         -------
         tuple[bool, str]
-            bool, True if the single bead input is valid, False otherwise.
-            string, Error message if the single bead input is invalid.
+            bool, True if the input is valid, False otherwise.
+            string, Error message if the input is invalid.
         """
+
         try:
-            test_machine = AdditiveMachine(
+            allowed_types = [
+                SimulationType.SINGLE_BEAD,
+                SimulationType.POROSITY,
+                SimulationType.MICROSTRUCTURE,
+            ]
+            if row[ColumnNames.TYPE] in allowed_types:
+                if row[ColumnNames.TYPE] == SimulationType.SINGLE_BEAD:
+                    if np.isnan(row[ColumnNames.START_ANGLE]):
+                        row[ColumnNames.START_ANGLE] = MachineConstants.DEFAULT_STARTING_LAYER_ANGLE
+                    if np.isnan(row[ColumnNames.ROTATION_ANGLE]):
+                        row[
+                            ColumnNames.ROTATION_ANGLE
+                        ] = MachineConstants.DEFAULT_LAYER_ROTATION_ANGLE
+                    if np.isnan(row[ColumnNames.HATCH_SPACING]):
+                        row[ColumnNames.HATCH_SPACING] = MachineConstants.DEFAULT_HATCH_SPACING
+                    if np.isnan(row[ColumnNames.STRIPE_WIDTH]):
+                        row[
+                            ColumnNames.STRIPE_WIDTH
+                        ] = MachineConstants.DEFAULT_SLICING_STRIPE_WIDTH
+            else:
+                return (False, f"Invalid simulation type: {row[ColumnNames.TYPE]}.")
+
+            machine = AdditiveMachine(
                 laser_power=row[ColumnNames.LASER_POWER],
                 scan_speed=row[ColumnNames.SCAN_SPEED],
                 heater_temperature=row[ColumnNames.HEATER_TEMPERATURE],
                 layer_thickness=row[ColumnNames.LAYER_THICKNESS],
                 beam_diameter=row[ColumnNames.BEAM_DIAMETER],
+                starting_layer_angle=row[ColumnNames.START_ANGLE],
```

Comment:
```suggestion
                starting_layer_angle=row[ColumnNames.START_ANGLE] if not np.isnan(row[ColumnNames.START_ANGLE]) else MachineConstants.DEFAULT_STARTING_LAYER_ANGLE
```
Repeat this for `layer_rotation_angle, hatch_spacing, slicing_stripe_width`.